### PR TITLE
In case of float type we use PDO::PARAM_STR to bind

### PR DESCRIPTION
**Pull request by emulgeator**

### CODE REVIEW

File: test/YapepBase/Mock/Database/DbConnectionMock.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ *
+ * @codeCoverageIgnore
+ */
+class DbConnectionMock extends DbConnection {
+
+	public function __construct() {
+	}
+
+	protected function connect(array $configuration) {
+		// TODO: Implement connect() method.
```
Comment (szeber) [inline test/YapepBase/Mock/Database/DbConnectionMock.php]: Please either implement or remove the todo :D

File: test/YapepBase/Mock/Database/DbConnectionMock.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	}
+
+	protected function getBackendType() {
+		// TODO: Implement getBackendType() method.
+	}
+
+	public function getParamType(&$value) {
```
Comment (szeber) [inline test/YapepBase/Mock/Database/DbConnectionMock.php]: Stupid question, but why is this needed? I mean, wouldn't it just work without this method being overridden?

Comment (emulgeator) [reply test/YapepBase/Mock/Database/DbConnectionMock.php]: Not really. The parent method is protected and I need to call it somehow to test separately.
Actually this method should belong to a another class but I didn't want to refact more now.
It will be fixed in the never coming next version anyway :)
